src/Dev/SapphireTest.php

```diff
@@ -1232,9 +1231,9 @@ public function logInWithPermission($permCode = "ADMIN")
                 $group->Permissions()->add($permission);
             }
 
-            $member = DataObject::get_one('SilverStripe\\Security\\Member', array(
-                '"Member"."Email"' => "$permCode@example.org"
-            ));
+            $member = Member::get()->filter([
```

Comment:
After each test `DataObject::singleton()->flushCache();` is called so the cache is discarded for get_ones anyway.